`motion-v` Vite error

**1. Describe the bug**
Adding `motion-v` to my project (replacing `motion`) results in this error:

```
  VITE v6.2.3  ready in 1122 ms

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
  ➜  Vue DevTools: Open http://localhost:5173/__devtools__/ as a separate window
  ➜  Vue DevTools: Press Alt(⌥)+Shift(⇧)+D in App to toggle the Vue DevTools
  ➜  press h + enter to show help
✘ [ERROR] No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "GroupPlaybackControls"

    ../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/interfaces/motion-value.mjs:1:36:
      1 │ import { getValueTransition, frame, GroupPlaybackControls } from "motion-dom";
        ╵                                     ~~~~~~~~~~~~~~~~~~~~~

✘ [ERROR] No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "GroupPlaybackControls"

    ../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/animate/index.mjs:1:9:
      1 │ import { GroupPlaybackControls } from "motion-dom";
        ╵          ~~~~~~~~~~~~~~~~~~~~~

✘ [ERROR] No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "NativeAnimationControls"

    ../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/animators/waapi/NativeAnimation.mjs:1:9:
      1 │ import { NativeAnimationControls, isGenerator, createGeneratorEasing, supportsLinearEasing } ...
        ╵          ~~~~~~~~~~~~~~~~~~~~~~~

✘ [ERROR] No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "GroupPlaybackControls"

    ../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/animators/waapi/animate-style.mjs:1:9:
      1 │ import { GroupPlaybackControls } from "motion-dom";
        ╵          ~~~~~~~~~~~~~~~~~~~~~

1472 |     let { file, line, column } = e.location;
1473 |     let pluginText = e.pluginName ? `[plugin: ${e.pluginName}] ` : "";
1474 |     return `
1475 | ${file}:${line}:${column}: ERROR: ${pluginText}${e.text}`;
1476 |   }).join("");
1477 |   let error = new Error(text);
                     ^
error: Build failed with 4 errors:
../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/animate/index.mjs:1:9: ERROR: No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "GroupPlaybackControls"
../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/animators/waapi/NativeAnimation.mjs:1:9: ERROR: No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "NativeAnimationControls"
../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/animators/waapi/animate-style.mjs:1:9: ERROR: No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "GroupPlaybackControls"
../../node_modules/motion-v/dist/es/external/.pnpm/framer-motion@12.5.0/external/framer-motion/dist/es/animation/interfaces/motion-value.mjs:1:36: ERROR: No matching export in "../../node_modules/motion-dom/dist/es/index.mjs" for import "GroupPlaybackControls"
      at failureErrorWithLog (/home/kainoa/Git/redacted/node_modules/esbuild/lib/main.js:1477:15)
      at <anonymous> (/home/kainoa/Git/redacted/node_modules/esbuild/lib/main.js:946:25)
      at <anonymous> (/home/kainoa/Git/redacted/node_modules/esbuild/lib/main.js:1355:9)
```


**Environment details**

`motion-v` 1.0.0-alpha.4
`vite` 6.2.3
`vue` 3.5.13

--- a/packages/motion/vite.config.ts
+++ b/packages/motion/vite.config.ts
@@ -48,7 +48,6 @@ export default defineConfig({
         'vue',
         'hey-listen',
         '@vueuse/core',
-        'motion-dom',
       ],
       output: [
         {

